src/snovault/local_storage.py

```diff
@@ -0,0 +1,133 @@
+import binascii
+
+from os import urandom
+from datetime import datetime
+from pytz import timezone
+
+from redis import StrictRedis
+from pyramid.view import view_config
+
+from snovault.interfaces import (
+    LOCAL_STORAGE,
+    LOCAL_STORAGE_EVENT_TAGS,
+    LOCAL_STORAGE_STATE,
+)
+
+
+def includeme(config):
+    config.scan(__name__)
+    config.add_route('local_storage', '/local_storage')
+    config.registry[LOCAL_STORAGE] = LocalStoreClient(
+        db=config.registry.settings['local_storage_db'],
+        host=config.registry.settings['local_storage_host'],
+        port=config.registry.settings['local_storage_port'],
+        timeout=int(config.registry.settings['local_storage_timeout']),
+    )
+
+
+@view_config(route_name='local_storage', request_method='GET')
+def local_storage(request):
+    '''Examples of how to store and get from redis'''
+    local_store = request.registry[LOCAL_STORAGE]
+    local_tz = request.registry.settings.get('local_tz')
+    result = {
+        'utc_now': str(datetime.utcnow()),
+        'lcl_now': f"{local_tz}: {datetime.now(timezone(local_tz))}",
+        # Different pyramid process may have different registries, this returns memory address
+        'local_storage': str(local_store),
+    }
+
+    # Set/Get dict or hash
+    redis_info = local_store.dict_get(LOCAL_STORAGE_STATE)
+    if not redis_info:
+        redis_info = {
+            'db': local_store.db,
+            'host': local_store.host,
+            'port': local_store.port,
+            'timeout': local_store.timeout,
+        }
+        local_store.dict_set(LOCAL_STORAGE_STATE, redis_info)
+        # Initial single item set/get example
+        local_store.item_set('counter', 0)
+    result['redis_info'] = redis_info
+
+    # Set/Get single item
+    result['counter'] = int(local_store.item_get('counter')) + 1
+    local_store.item_set('counter', result['counter'])
+
+    # Store event key/value pairs with tag and list
+    event_key = request.params.get("event_key")
+    event_value = request.params.get("event_value")
+    event_tag = request.params.get("event_tag")
+    if event_key and event_value:
+        # Key values pairs are tagged for easier retrieval/organization
```

Comment:
I think we should split this endpoint into two:
GET /events => get event info by tag or all events
POST /events => create a new event from a pair (event_key, event_value)
or any similar name for the endpoints :)

Reply:
these were moved SNO-173, this PR doesn't have an implementation of the local store.  It seemed like unnecessary code.